suffuse/node_fuse.go

```diff
@@ -3,22 +3,92 @@ package suffuse
 import (
   "github.com/suffuse/go-suffuse/suffuse/xattr"
   "os"
+  "time"
   "golang.org/x/net/context"
   "bazil.org/fuse/fs"
   f "bazil.org/fuse"
 )
 
+// Access checks whether the calling context has permission for
```

Comment:
But more importantly I don't want us off on some side trip writing go-language formatting software. I want to remain very focused on the actual thing. I am confident in our ability to go back and trim whitespace and change comments around down the road. And no this doesn't mean throwing caution to the wind, it just means I do not think we can possibly come out ahead by being especially concerned about cosmetic issues at this juncture.